lib/ide-ignore.txt

```diff
@@ -22,5 +21,6 @@ tmp/lib-pre/com.liferay.petra.string.jar
 tmp/lib-pre/com.liferay.petra.test.util.jar
 tmp/lib-pre/com.liferay.petra.url.pattern.mapper.jar
 tmp/lib-pre/com.liferay.petra.xml.jar
+tmp/lib-pre/com.liferay.portal.encryptor.jar
```

Comment:
@julschong Do we need to put "com.liferay.portal.encryptor.jar" here after it is moved to as a normal module.

Tina.